package.json

```diff
@@ -2,8 +2,8 @@
   "name": "corporate-components",
   "version": "1.0.0",
   "description": "Component library for main Wellcome corporate site",
-  "main": "dist/esm/index.js",
-  "module": "dist/esm/index.js",
+  "main": "dist/index.js",
```

Comment:
I think this is probably fine as we'll be using the library internally as ES modules.

FYI, the reason for `esm` was in the case we would want to export in **commonjs** AND **es modules**. A properly setup library (for public consumption) prob should cater for both with a package.json configured in similar way to the following:

```
// package.json
...
  "main": "dist/index.js",
  "module": "dist/esm/index.js",
...
```

Reply:
Ah, thanks for flagging. Now I'm looking at it, I'm not sure why I removed the `module` value... 😬

I updated the `main` as the postcss plugin dumps the compiled CSS to wherever the `main` entry is. I could set the `entry` option on the postcss plugin in the rollup config, though, and we could keep the `esm` folder?

Reply:
Ok cool. Let's keep `esm` to make it a little clearer that that's what's available. We can add tooling to export commonjs too in future if we think it's needed